ConcernsCaseWork/ConcernsCaseWork.Tests/Pages/Case/Management/Action/Nti/AddPageModelTests.cs

```diff
@@ -0,0 +1,114 @@
+﻿using ConcernsCaseWork.Pages.Case.Management.Action.Nti;
+using ConcernsCaseWork.Services.Cases;
+using ConcernsCaseWork.Services.Nti;
+using ConcernsCaseWork.Shared.Tests.Factory;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.RazorPages;
+using Microsoft.AspNetCore.Mvc.Routing;
+using Microsoft.AspNetCore.Mvc.ViewFeatures;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Primitives;
+using Moq;
+using NUnit.Framework;
+using Service.Redis.Nti;
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace ConcernsCaseWork.Tests.Pages.Case.Management.Action.Nti
```

Comment:
There's quite a lot of logic in the Add page - could do with some more tests maybe?